### ISSUE TITLE
need test for InclusionWait > 0 && tx is refused

### ISSUE THREAD

**Issue by jeffallen** [dedis/cothority#1431]
This comment says that we probably don't notify correctly:

```
	for _, ct := range body.Transactions {
		// TODO: Where is the other call, for informWaitChannel(hash, false)?
		s.state.informWaitChannel(ct.Instructions.Hash(), true)
	}
```

Need a test to figure this out and solve it.

**Comment by jeffallen** [comment]
This will be fixed in #1437.

**Comment by jeffallen** [comment]
Fixed.
